docs/readmes/feg/deploy_build.md

```diff
@@ -6,15 +6,38 @@ hide_title: true
 
 # Building Federation Gateway Components
 
-Start up your Docker daemon, `cd` to where you've cloned Magma, then:
+If you cloned Magma using git, make sure you are checked out on the release you
+intend to build.
+
+In case you need to change the version you can:
+
+```bash
+# to list all releases
+git tag -l
+
+# to switch to a different release (for example v1.3.3)
+git checkout v1.3.3
+
+# to switch to master (developement version)
+git checkout master
+```
+
+Once you are on a proper version of Magma, make sure your Docker deamon is running.
+Then go run those commands to build FeG.
 
 ```bash
 cd magma/feg/gateway/docker
 docker-compose build --parallel
 ```
 
-If this is your first time building the FeG, this may take a while. When this
-job finishes, upload these images to your image registry:
+Note that you are building FeG from your local repository. There is no need to
+change content `.env`
+
+If this is your first time building the FeG, this may take a while.
+
+When this job finishes, you will have built FeG on your local machine. In case
```

Comment:
minor nit: How does one verify that FeG is built, what are the names of the built docker images ?

Reply:
Can you also provide more info on what services map to what docker image ?

Reply:
I prefer not to add too much unnecessary information in this guide that can distract. 

To start feg you need all images on your registry. I don't think knowing what image each service use can help in this step. The information per service can be seen from docker-compose

But let me check if i cant remove the need of radius image in case cwf is not used